<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Output File(s)</string>
        </property>
        <property name="buddy">
         <cstring>outputFilesLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="toolButton_2">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="inputFileLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="outputFilesLineEdit">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input File</string>
        </property>
        <property name="buddy">
         <cstring>inputFileLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="transposeCheckBox">
         <property name="text">
          <string>Transpose matrix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fileForEachRowCheckBox">
         <property name="text">
          <string>Create a file for each output row</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Characters to be replaced by numbers in the output file name</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>replaceCharsLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="replaceCharsLineEdit">
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Convert</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>inputFileLineEdit</tabstop>
  <tabstop>toolButton_2</tabstop>
  <tabstop>outputFilesLineEdit</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>transposeCheckBox</tabstop>
  <tabstop>fileForEachRowCheckBox</tabstop>
  <tabstop>replaceCharsLineEdit</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>fileForEachRowCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectInputFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectOutputFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>runConversion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectInputFile()</slot>
  <slot>selectOutputFiles()</slot>
  <slot>runConversion()</slot>
 </slots>
</ui>
